hass-addon-sunsynk-dev/run.py

```diff
@@ -107,6 +167,7 @@ def _handler(value):
             continue
 
         if isinstance(sensor, TimeRWSensor):
+            ent.update({"icon": "mdi:clock"})
```

Comment:
Here you can simply include icon in the parameters to SelectEntity a couple lines down...

Reply:
Unfortunately not because `icon` is already set on `ent` in the `RWSensor` branch and I got an error stating that I am supplying `icon` twice - once with the `ent` dictionary and once as an explicit field when I tried.